packages/test-studio/schemas/previewMediaTest.js

```diff
@@ -0,0 +1,29 @@
+export default {
+  name: 'previewMediaTest',
+  type: 'document',
+  title: 'Preview: Using media in preview',
+  preview: {
+    select: {
+      title: 'title',
+      media: 'image'
+    },
+  },
+  fields: [
+    {
+      name: 'title',
+      title: 'Title',
+      type: 'string'
+    },
+    {
+      name: 'image',
+      title: 'Image',
+      type: 'image'
```

Comment:
Maybe add `options: {hotspot: true}` here since it is an important part of preview here.

Reply:
Good point!